src/Raven.Server/Documents/PeriodicBackup/DirectUpload/AwsS3DirectUploadStream.cs

```diff
@@ -0,0 +1,30 @@
+using Raven.Server.Documents.PeriodicBackup.Aws;
+using Raven.Server.Documents.PeriodicBackup.Retention;
+using Sparrow;
+
+namespace Raven.Server.Documents.PeriodicBackup.DirectUpload;
+
+public class AwsS3DirectUploadStream : DirectUploadStream<RavenAwsS3Client>
+{
+    private readonly RetentionPolicyBaseParameters _retentionPolicyParameters;
+
+    protected override long MaxPartSizeInBytes { get; }
+
+    public AwsS3DirectUploadStream(Parameters parameters) : base(parameters)
+    {
+        _retentionPolicyParameters = parameters.RetentionPolicyParameters;
+
+        MaxPartSizeInBytes = Client.MinOnePartUploadSizeLimit.GetValue(SizeUnit.Bytes);
```

Comment:
I am a little bit confused by these variable names...

Should we change it like this?
```suggestion
        MaxPartSizeInBytes = Client.MaxUploadPutObject.GetValue(SizeUnit.Bytes);
```

Reply:
It should be `MinOncePartUploadSizeInBytes`